roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -176,3 +176,12 @@ memcached_servers = {{ hostvars|ursula_memcache_hosts(groups, memcached.port) }}
 allow_headers = X-Auth-Token,X-Identity-Status,X-Roles,X-Service-Catalog,X-User-Id,X-Tenant-Id,X-OpenStack-Request-ID,X-Trace-Info,X-Trace-HMAC,OpenStack-Volume-microversion
 expose_headers = X-Auth-Token,X-Subject-Token,X-Service-Token,X-OpenStack-Request-ID,OpenStack-Volume-microversion
 allow_methods = GET,PUT,POST,DELETE,PATCH
+
+{% if barbican.enabled|bool %}
+[key_manager]
+api_class = castellan.key_manager.barbican_key_manager.BarbicanKeyManager
+
+[barbican]
+barbican_endpoint = {{ endpoints.barbican.url.public }}
+auth_endpoint = {{ endpoints.keystone.url.public }}/{{ endpoints.barbican.keystone_version }}
```

Comment:
we do not need new variable to be defined i.e `endpoints.barbican.keystone_version` is not needed.
use `endpoints.keystonev3.version`